webrtc.html

```diff
@@ -7338,6 +7355,81 @@ <h2>Methods</h2>
+    the <code>addTrack</code> method cannot provide simulcast functionality since it
+    does not take <code>sendEncodings</code> as an argument, and therefore cannot configure
+    an <code><a>RTCRtpTransceiver</a></code> with a "simulcast envelope".</p>
+    <p>While <code>setParameters</code> cannot modify "simulcast envelope", it is still possible to control the
+    number of streams that are sent and the characteristics of those streams.
+    Using <code>setParameters</code>, simulcast streams can be made inactive by setting the <code>active</code>
+    attribute to <code>false</code>, or can be reactivated by setting the <code>active</code>
+    attribute to <code>true</code>.  Using <code>setParameters</code>, stream characteristics can be changed 
+    by modifying attributes such as <code>maxBitrate</code> and <code>maxFramerate</code>.</p>
+    <p>This specification does not define how to configure <code>createOffer</code> to
+    receive multiple RTP encodings. However when <code>setRemoteDescription</code> is called with a
+    corresponding remote description that is able to send
+    multiple RTP encodings as defined in [[!JSEP]], the
+    <code><a>RTCRtpReceiver</a></code> may receive multiple RTP
+    encodings and the parameters retrieved via the transceiver's
+    <code>receiver.getParameters()</code> will reflect the
+    encodings negotiated.</p>
+    <p class="note">An <code><a>RTCRtpReceiver</a></code> can receive multiple RTP streams in a scenario
+    where a Selective Forwarding Unit (SFU) switches between simulcast streams it receives from user agents.
+    If the SFU does not rewrite RTP headers so as to arrange the switched streams into a single RTP
+    stream prior to forwarding, the <code><a>RTCRtpReceiver</a></code> will receive packets from
+    distinct RTP streams, each with their own SSRC and sequence number space.
+    While the SFU may only forward a single RTP stream at any given time, due to reordering packets
+    from multiple RTP streams can become intermingled at the receiver.
+    An <code><a>RTCRtpReceiver</a></code> equipped to receive multiple RTP streams will therefore
+    need to be able to correctly order the received packets, recognize potential loss events and
+    react to them. Correct operation in such a situation is non-trivial and therefore is optional
+    for implementations of this specification.</p>
+   <section class="informative" id="rtcrtpencodingspatialsim-example*">
+   <h3>Encoding Parameter Examples</h3>
+   <div>
+     <p>Below are examples of simulcast usage scenarios implemented with encoding parameters.</p>
+   <pre class="example highlight">
+   // Example of 3-layer spatial simulcast
+var encodings = [{
+  // Simulcast layer at full scale
+  rid: "f",
+  scaleResolutionDownBy: 1.0
+}, {
+  // Simulcast layer at one half scale
+  rid: "h",
+  scaleResolutionDownBy: 2.0
+}, {
+  // Simulcast layer at one quarter scale
+  rid: "q",
+  scaleResolutionDownBy: 4.0
+}];
```

Comment:
This example looks identical to [10.4 Simulcast Example](http://w3c.github.io/webrtc-pc/#simulcast-example). Do we need both?

Would it suffice to decorate the existing example with these comments?

Reply:
Deleted the overlapping example and added a different one.